### ISSUE TITLE
Is there any example with hooks

### ISSUE THREAD

**Issue by thug1705** [jakezatecky/react-checkbox-tree#198]
Hi,

Is there any basic example that use react hooks ? 

Cheers

**Comment by jakezatecky** [comment]
Sorry, no example for now. I have not used Hooks myself, but I imagine the component should not have any issue using them.

**Comment by olessiap** [comment]
@thug1705 did you get this example to work with hooks by chance?

**Comment by thug1705** [comment]
> @thug1705 did you get this example to work with hooks by chance?

@olessiap yes I but I could make it work as expected with version "1.5.0"

**Comment by olessiap** [comment]
@thug1705 i made it work with hooks! let me know if you need any assistance

**Comment by brandonam** [comment]
@olessiap have you had a chance to use the filter with hooks?

**Comment by olessiap** [comment]
@brandonam yes! I made filters work with hooks. The only issue is now the bug where searched for items automatically get checked. Looks like somebody already submitted a fix for this [here,](https://github.com/jakezatecky/react-checkbox-tree/pull/202), so hopefully it gets merged soon!! @brandonam let me know if you need anything with implementing hooks :)

**Comment by FrancoisYu** [comment]
> @thug1705 i made it work with hooks! let me know if you need any assistance

It would be very appreciated if you can share an example code.  Thanks.

**Comment by olessiap** [comment]
@FrancoisYu, you can see an example of fileters with hooks [here](https://github.com/olessiap/metric-dashboard-demo/blob/master/src/components/modals/RouterModal.js)!
